library/src/main/java/xmtpv3.kt

```diff
@@ -413,6 +413,8 @@ internal interface _UniFFILib : Library {
     ): RustBuffer.ByValue
```

Comment:
There should have been a `libxmtp-version.txt` generated alongside the `.kt` definition, could we check that in too?